packages/esm-form-entry-app/src/app/form-schema/form-schema.service.ts

```diff
@@ -6,18 +6,31 @@ import { FormResourceService } from '../openmrs-api/form-resource.service';
 import { FormSchemaCompiler } from '@openmrs/ngx-formentry';
 import { LocalStorageService } from '../local-storage/local-storage.service';
 import { FormSchema, Questions } from '../types';
+import { TranslateService } from '@ngx-translate/core';
 
 @Injectable()
 export class FormSchemaService {
   constructor(
     private formsResourceService: FormResourceService,
     private localStorage: LocalStorageService,
     private formSchemaCompiler: FormSchemaCompiler,
+    private translate: TranslateService,
   ) {}
 
   public getFormSchemaByUuid(formUuid: string, cached: boolean = true): Observable<FormSchema> {
     const cachedCompiledSchema = this.getCachedCompiledSchemaByUuid(formUuid);
+    const currentLang: string = (window as any).i18next.language.substring(0, 2).toLowerCase() || 'en';
```

Comment:
Do we need to be so explicit regarding the type (`string`) here? Wouldn't it be better to type `window.i18next` (and make it `i18n | undefined`, such that `window.i18next?.language...` would be a thing).

Reply:
Thanks @FlorianRappl I will change that.

Reply:
@FlorianRappl I don't understand the suggested change here, is there a type name `i18n`?